build.sbt

```diff
@@ -726,6 +726,20 @@ lazy val `http4s-swagger` = projectMatrix
   )
   .http4sJvmPlatform(allJvmScalaVersions, jvmDimSettings)
 
+lazy val cats = projectMatrix
```

Comment:
don't forget to add it to the aggregate. Because the build is green, I thought the tests were actually run and that informed the way I approached the PR.